homeassistant/components/binary_sensor/rainmachine.py

```diff
@@ -20,7 +20,10 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-def setup_platform(hass, config, add_devices, discovery_info=None):
+async def async_setup_platform(hass,
```

Comment:
Please break line after the parenthesis instead,and fill out as much as possible of the line. That's more in line with home assistant style.

Reply:
FYI, I found a lot of other style inconsistencies based on your above request, so I went ahead and addressed them.